USER_MANUAL.md

```diff
@@ -44,7 +44,14 @@ Actors used in gyms are in `Contetnt\Actors`: add any new Actors to this directo
 
 ##### Handover gym
 * Demonstrates that an entity can cross from one area of authority to another.
-* Contains a cube that moves back and forth across a floor.
+* Contains a set of cube that moves back and forth across a floor.
```

Comment:
nit - "a set of cubes" or "four cubes" that "move".

Reply:
will do